src/OctoshiftCLI.Tests/ado2gh/Services/AdoInspectorServiceTests.cs

```diff
@@ -120,5 +120,53 @@ public async Task GetPipelines_Should_Return_All_Pipelines()
             // Assert
             result.Should().BeEquivalentTo(pipelines);
         }
+
```

Comment:
Shouldn't we also add tests for failure scenarios? i.e. garbage csv, etc.

Reply:
I don't think it's necessary. I'm relying on TextFieldParser to parse the CSV, and I don't want to write tests for dotnet functionality. So if the csv is utter garbage TextFieldParser will error out. And anything else we'll treat as org/team-project/repo.